src/WebJobs.Script.WebHost/Management/LinuxSpecialization/RunFromPackageHandler.cs

```diff
@@ -193,10 +193,10 @@ public async Task<bool> MountAzureFileShare(HostAssignmentContext assignmentCont
                 using (_metricsLogger.LatencyEvent(MetricEventNames.LinuxContainerSpecializationMountCifs))
                 {
                     var targetPath = _environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteHomePath);
-                    _logger.LogDebug($"Mounting {EnvironmentSettingNames.AzureFilesContentShare} at {targetPath}");
+                    _logger.LogDebug("Mounting {azureFilesContentShare} at {targetPath}", EnvironmentSettingNames.AzureFilesContentShare, targetPath);
                     bool succeeded = await _meshServiceClient.MountCifs(assignmentContext.AzureFilesConnectionString,
                         assignmentContext.AzureFilesContentShare, targetPath);
-                    _logger.LogInformation($"Mounted {EnvironmentSettingNames.AzureFilesContentShare} at {targetPath} Success = {succeeded}");
+                    _logger.LogInformation("Mounted {azureFilesContentShare} at {targetPath} Success = {succeeded}", EnvironmentSettingNames.AzureFilesContentShare, targetPath, succeeded);
```

Comment:
`EnvironmentSettingNames.AzureFilesContentShare` is a constant. Not sure if we want that to be parameterized